src/sugar3/speech.py

```diff
@@ -220,6 +222,28 @@ def __init__(self, **kwargs):
         self._save_timeout_id = -1
         self.restore()
 
+    def _marked_messages_cb(self, player, value):
```

Comment:
Move this method below `_update_state` so that overall method order is consistent with the order in which it was mentioned.

Reply:
Will fix that, thanks.